code/modules/projectiles/projectile/energy/thermal.dm

```diff
@@ -10,10 +10,10 @@
 	bare_wound_bonus = 10
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser
 
-/obj/projectile/energy/inferno/on_hit(atom/target, blocked, pierce_hit)
-	..()
+/obj/projectile/energy/inferno/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
 	if(!ishuman(target))
-		return
+		return BULLET_ACT_HIT
```

Comment:
are we sure we want to override the return value?

Reply:
I'm not as sure with this one lemme revert that particular change.